scripts/analytics/node_summary.py

```diff
@@ -0,0 +1,102 @@
+        super(NodeSummary, self).get_events(date)
+
+        # Convert to a datetime at midnight for queries and the timestamp
+        timestamp_datetime = datetime(date.year, date.month, date.day)
+        query_datetime = timestamp_datetime + timedelta(1)
+
+        node_query = (
+            Q('is_deleted', 'ne', True) &
+            Q('is_folder', 'ne', True) &
+            Q('date_created', 'lt', query_datetime)
+        )
+
+        registration_query = node_query & Q('is_registration', 'eq', True)
+        non_registration_query = node_query & Q('is_registration', 'eq', False)
+        project_query = non_registration_query & Q('parent_node', 'eq', None)
+        registered_project_query = registration_query & Q('parent_node', 'eq', None)
+        public_query = Q('is_public', 'eq', True)
+        private_query = Q('is_public', 'eq', False)
+        retracted_query = Q('retraction', 'ne', None)
+        project_public_query = project_query & public_query
+        project_private_query = project_query & private_query
+        node_public_query = non_registration_query & public_query
+        node_private_query = non_registration_query & private_query
+        registered_node_public_query = registration_query & public_query
+        registered_node_private_query = registration_query & private_query
+        registered_node_retracted_query = registration_query & retracted_query
+        registered_project_public_query = registered_project_query & public_query
+        registered_project_private_query = registered_project_query & private_query
+        registered_project_retracted_query = registered_project_query & retracted_query
+
+        totals = {
+            'keen': {
+                'timestamp': timestamp_datetime.isoformat()
+            },
+            'nodes': {
+                'total': Node.find(node_query).count(),
+                'public': Node.find(node_public_query).count(),
+                'private': Node.find(node_private_query).count()
+            },
+            'projects': {
+                'total': Node.find(project_query).count(),
+                'public': Node.find(project_public_query).count(),
+                'private': Node.find(project_private_query).count(),
+            },
+            'registered_nodes': {
+                'total': Node.find(registration_query).count(),
+                'public': Node.find(registered_node_public_query).count(),
+                'embargoed': Node.find(registered_node_private_query).count(),
+                'withdrawn': Node.find(registered_node_retracted_query).count(),
+            },
+            'registered_projects': {
+                'total': Node.find(registered_project_query).count(),
+                'public': Node.find(registered_project_public_query).count(),
+                'embargoed': Node.find(registered_project_private_query).count(),
+                'withdrawn': Node.find(registered_project_retracted_query).count(),
+            }
+        }
```

Comment:
For the moment, I think we'll chalk this up to weird data in your local env, @felliott. If we find strangeness on production, we can see what we can find there as well, but it's not like the existing analytics don't have their share of oddities. 

If you want to do a quick pass of the timezone changes we can hopefully get this PR moved along and into Prod.